Software/Human-Zombie-Device/ArduinoBle_Manager.h

```diff
@@ -0,0 +1,68 @@
+#ifndef ARDUINOBLE_MANAGER_H
+#define ARDUINOBLE_MANAGER_H
+
+// External Arduino LIB.
+#include <ArduinoBLE.h>
+
+#include <vector>
+
+// Internal ZTag libs for Device Understanding
+#include "ReceivedPacket.h"
+
+class ArduinoBle_Manager
+{
+private:
+
+    // Had to make this static because it's used in the callback
+    // which has to be static because otherwise I couldn't get setEventHandler to compile with it
+    // There may be a better way, if there is i'd love to know it.
+    static std::vector<ReceivedPacket> foundZtagDevices;
```

Comment:
Addressing the comment of "There may be a better way" - > My question would be, why would there need to be. 

This is being hit, because the class is currently set up as something that could have mulitple instances of itself, 

My question would be, are there any instances you can think about there you would need multiple instances of ArduinoBle_Manager ?

Otherwise, https://en.wikipedia.org/wiki/Singleton_pattern - 
https://stackoverflow.com/questions/1008019/c-singleton-design-pattern

Unless, I have the wrong handle on this :) 

At some point, your newfangled C++ is going to pass what I know, and then we may have to disturb @MikeS159